css/fullcalendar.scss

```diff
@@ -171,3 +171,24 @@
 .fc * {
 	box-sizing: content-box !important;
 }
+
+// Reminder icon on events with alarms set
+.fc-event-nc-alarms {
+	.icon-event-reminder {
+		background-color: inherit;
+		background-position: right;
+		position: absolute;
+		top: 0;
+		right: 0;
+	}
+	.icon-event-reminder-light {
```

Comment:
Nitpick: Can you please change this to [BEM style](https://css-tricks.com/bem-101/).

Then you can turn `.icon-event-reminder-light` to `.icon-event-reminder--light` and using SCSS, you can include it in the `.icon-event-reminder {` block as `&--light`.